### ISSUE TITLE
[Suggestion] Add Shutter Encoder to Chocolatey Package Manager

### ISSUE THREAD

**Issue by pikasalt** [paulpacifico/shutter-encoder#149]
It would be handy to have this application in chocolatey, since it would reduce the time spent opening the application and having to deal with a notification popup. Fantastic application!

**Comment by Manrochri** [comment]
I totally agree. It's, in my humble opinion, the best software for doing multimedia file conversions

**Comment by bjspi** [comment]
Yes that would be awesome. Best of both works!:)

**Comment by iamLazyCode** [comment]
any update on this ?

**Comment by kiquenet** [comment]
updates 2025 ?
